mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx

```diff
@@ -0,0 +1,11 @@
+import styles from "src/pages/pageAllWays/AllWaysPage.module.scss";
+/**
+ * Page contains ways to all pages
```

Comment:
It's wrong comment/. It is not page for all pages...

Reply:
done